sources/packages/backend/libs/utilities/src/config/config.service.ts

```diff
@@ -253,4 +253,14 @@ export class ConfigService {
   get apiVersion(): string {
     return this.getCachedConfig("apiVersionConfig", process.env.VERSION);
   }
+
+  /**
+   * Load test API port.
+   */
+  get loadTestAPIPort(): number {
```

Comment:
Since it is not a production code, should we keep this, and possible others in the future, together with the production ones?

Reply:
May be it is not worth it. I can use a direct .env ref and default value to it.